services/movie.py

```diff
@@ -0,0 +1,44 @@
+from django.db.models import QuerySet
+
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None

```

Comment:
Accepts Optional[list[int]]
If default parameter is None the typing must include optional
Change in other places